scripts/local_setup.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# This script is used to run the harmony backend locally.
+# It consist of 5 arguments which are:
+# 1. The environment file to be used
+# 2. The email of the user to be created
+# 3. The first name of the user to be created
+# 4. The last name of the user to be created
+# 5. The password of the user to be created
+
+# Function to check if all required arguments are provided
+check_arguments() {
+    if [ $# -ne 5 ]; then
+        echo "Error: Missing arguments."
+        echo "Usage: $0 <env_file> <email> <first_name> <last_name> <password>"
+        exit 1
+    fi
+}
+
+# Function to install and configure the virtual environment
+setup_virtualenv() {
+    echo "Creating virtual environment..."
+    pwd
+    python3.9 -m venv venv
```

Comment:
I think you misunderstood my first request - we still want to have `python3.9 -m venv venv` inside a ` if [ ! -d "venv" ]; then`

Reply:
Sorry, I misunderstood the request. I will correct this.